google_takeout_parser/parse_json.py

```diff
@@ -149,12 +152,16 @@ def _parse_location_history(p: Path) -> Iterator[Res[Location]]:
         yield RuntimeError(f"Locations: no 'locations' key in '{p}'")
     for loc in json_data.get("locations", []):
         accuracy = loc.get("accuracy")
+        deviceTag = loc.get("deviceTag")
+        source = loc.get("source")
         try:
             yield Location(
                 lng=float(loc["longitudeE7"]) / 1e7,
                 lat=float(loc["latitudeE7"]) / 1e7,
                 dt=_parse_timestamp_key(loc, "timestamp"),
                 accuracy=None if accuracy is None else float(accuracy),
+                deviceTag=None if deviceTag is None else deviceTag,
+                source=None if source is None else str(source)
```

Comment:
is there a reason to convert it to str? in my case these are already str?

Reply:
no specific reason I think I was just blindly copying how it was being done with accuracy, so I can remove it. Although I will update the deviceTag to be cast to int because with orjson I think it returns it in scientific notation (ex. 8.024144696862913e+19)  if it isn't cast.